packages/e2e-tests/fixtures/blocks/core__table-of-contents.json

```diff
@@ -1,12 +1,12 @@
 [
-	{
-		"clientId": "_clientId_0",
-		"name": "core/table-of-contents",
-		"isValid": true,
-		"attributes": {
-			"onlyIncludeCurrentPage": false
-		},
-		"innerBlocks": [],
-		"originalContent": ""
-	}
+    {
```

Comment:
These changes in `table-of-contents` are irrelevant and are a result of regenerating the fixtures. If this PR delay to land, I'll handle it in a separate PR.